incubator/hnc/scripts/kind-watch

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+pod=`kubectl get pods -n hnc-system -ojsonpath='{.items[0].metadata.name}'`
```

Comment:
Protip, you can use `kubectl logs` on a deployment so this step is easier.

Reply:
D'oh, fixed. Thanks.